src/domexpression.js

```diff
@@ -31,6 +34,17 @@ var _ = Mavo.DOMExpression = $.Class({
 			this.expression = expression;
 		}
 
+		if (this.attribute?.startsWith("mv-attr-")) {
```

Comment:
It would be great to refactor this into a helper function since it's used multple times in this file, e.g.:

```
const MAVO_ATTR_PREFIX = 'mv-attr-';

function isMavoAttribute(attribute) {
     return attribute.startsWith(MAVO_ATTR_PREFIX);
}
```

Reply:
Might be worth having a constant with the prefix (marginally, so I'll leave it up to @DmitrySharabin ), but having a helper function to avoid two `startsWith()` calls is the definition of [premature abstraction](https://medium.com/@thisdotmedia/the-cost-of-premature-abstraction-b5d71ffd6400). In general, [duplication is cheaper than the wrong abstraction](https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction), so I tend to follow (and recommend) the [rule of three strikes](https://en.wikipedia.org/wiki/Rule_of_three_(computer_programming)).

Reply:
Honestly, I couldn’t find a good place for a constant holding the attribute name, so I decided to stick with the approach we already have for now. Moreover, it fits nicely with the rest of the code. If someday we decide to change the attribute name and refactor the code, I believe Find/Replace will quickly get the job done. 😄